src/main/java/com/team1/hrbank/domain/department/controller/DepartmentController.java

```diff
@@ -64,4 +65,11 @@ public ResponseEntity<DepartmentPageResponseDto> findDepartments(
     return ResponseEntity.ok(responseDto);
   }
 
+  @DeleteMapping(path = "{id}")
+  public ResponseEntity<DepartmentDto> delete(@PathVariable("id") Long id
```

Comment:
```suggestion
  public ResponseEntity<Void> delete(@PathVariable("id") Long id
```
실제로 반환되는 응답은 없기 때문에 Void로 바꿔주시면 감사하겠습니다!